algorithms/Algorithms_Answers.md

```diff
@@ -1 +1,47 @@
-Add your answers to the Algorithms exercises here.
\ No newline at end of file
+Add your answers to the Algorithms exercises here.
+
+#Excercise I.
+
+a.) Linear? The bigger n is, the bigger the loop grows. ```n*n*n``` is still n runtime.
+
+b.) log n. i is being split in half, then looping back through to check a value that is plit in 
+half; only to be split in half again.
+
+c.) First for loop would be log n/ 2, but then second is n, and 3rd is n. Time is as big as the biggest runtime so it would probably be ```n**2/ n ^ 2```.
```

Comment:
This one will end up being O(sqrt(n)) since the outer for loop grows with the size of √n.

The two inner for loops are considered constant since they don't grow with the input size of `n` instead they _always_ iterate to 8.